src/visualization/types/SimpleTable/PagedTable.tsx

```diff
@@ -274,8 +299,8 @@ const PagedTable: FC<Props> = ({result, properties}) => {
   }, []) // eslint-disable-line react-hooks/exhaustive-deps
 
   const size = useMemo(() => {
-    return measurePage(result, offset, height)
-  }, [result, offset, height])
+    return measurePage(result, offset, height, headerHeight, rowHeight)
+  }, [result, offset, height, headerHeight, rowHeight])
```

Comment:
should `measurePage` be included in the dependency array?

Reply:
No, the linter doesn't complain about `measurePage`. There is only one `measurePage` in the whole module, as it is a constant. That's probably the reason why the linter doesn't complain about it.